src/main/java/com/project/momo/service/MemberService.java

```diff
@@ -0,0 +1,112 @@
+package com.project.momo.service;
+
+import com.project.momo.common.utils.AuthUtils;
+import com.project.momo.dto.member.MemberInfoResponse;
+import com.project.momo.dto.member.PasswordUpdateRequest;
+import com.project.momo.dto.payment.PaymentRequest;
+import com.project.momo.entity.Member;
+import com.project.momo.entity.Payment;
+import com.project.momo.repository.MemberRepository;
+import com.project.momo.repository.PaymentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+    private final PaymentRepository paymentRepository;
+    private final PasswordEncoder passwordEncoder;
+
+    private final short MAX_PAYMENT_CNT = 3;
+
+    @Transactional(readOnly = true)
+    public MemberInfoResponse getMemberInfo() {
+        Optional<Member> member = memberRepository.findById(AuthUtils.getMemberId());
+        if (member.isEmpty()) {
+            throw new RuntimeException(); //TODO 예외 추가하기
+        }
```

Comment:
```suggestion
        Member member = memberRepository.findById(AuthUtils.getMemberId())
            .orElseThrow(() -> new RuntimeException("~~));
```
가 조금 더 올바른 Optional 사용법일 것 같네요

Reply:
네 정말 좋은 코드라고 생각합니다. Optional 사용법을 좀 더 공부해야겠다고 느꼈습니다ㅜㅜ